mock-2/public/index.html

```diff
@@ -0,0 +1,129 @@
+        </article>
+
+        <article class="topics-list__item topic topic_clickable">
+          <img class="topic__image" src="assets/topic__image_image_7.png" alt="clean dry road between snow">
+          <span class="topic__type">Lifestyle</span>
+          <h3 class="topic__title">Cold winter days</h3>
+          <p class="topic__text topic__text_short">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem.</p>
+        </article>
+      </div>
+    </section>
+
+    <section class="section section_centering-content section_vertical-field_tiny">
+      <button class="button">
+        load more
+      </button>
+    </section>
+
+    <div class="empty-space-filler"></div>
+
+    <footer class="section section_gray section_last section_vertical-field_tiny section_horizontal-field_regular">
+      <div class="footer-links">
+        <a class="footer-links__link">Terms and conditions</a>
+        <a class="footer-links__link">Privacy</a>
+
+        <span class="empty-space-filler"></span>
+
+        <a class="footer-links__link">Follow</a>
```

Comment:
Don't think this is a link, looks like it's just some kind of label.

And I think that social block could be a separate block, not an element of the footer because it could be placed somewhere else on the site later.